doc/wallet-backup.md

```diff
@@ -36,7 +36,27 @@ If you generate a new Zcash address, it will not be reflected in the backup file
 
 If your original `wallet.dat` file becomes inaccessible for whatever reason, you can use your backup by copying it into your data directory and renaming the copy to `wallet.dat`.
 
-### Using `z_exportwallet` & `z_importwallet`
+### Using `dumpprivkey` & `importprivkey`
+
+For exporting a single private key for a transparent address, you can use the command inherited from Bitcoin:
+
+`zcash-cli dumpprivkey <t-address>`
+
+This will return the private key and will not create a new file.
+
+To import a private key for a transparent address, use:
+
+`zcash-cli importprivkey <t-priv-key>`
+
+See the command's help documentation for instructions on fine-tuning the wallet rescan:
+
+`zcash-cli help importprivkey`
```

Comment:
zen-cli

Reply:
Thanks, I've updated all the occurrences!